package.json

```diff
@@ -54,14 +52,11 @@
     "fs-extra": "9.0.1",
     "glob": "^7.1.6",
     "got": "11.8.1",
-    "ignore-loader": "0.1.2",
     "imagemin": "7.0.1",
     "imagemin-gifsicle": "7.0.0",
-    "imagemin-jpegtran": "7.0.0",
     "imagemin-mozjpeg": "9.0.0",
```

Comment:
This is weird. Why `imagemin-jpegtran` but not `imagemin-gifsicle`?

Reply:
That one is referenced here https://github.com/mdn/yari/blob/c63505f68dedda244cb6007b101c63f9fb8a3ae3/filecheck/checker.js#L9-L13

Reply:
Oh I see. We're not even using `imagemin-jpegtran` at all. Independent of it being mentioned in the wrong place in the package.json.